### PR TITLE
Update `--snapshot-count` to reflect new reduced default for 3.6+ and include new tls min/max

### PR DESCRIPTION
**Pull request by jmhbnz**
1) A quick docs follow-up after https://github.com/etcd-io/etcd/pull/15408. For 3.6+ the default `--snapshot-count` has been reduced to 10,000.

Refer https://github.com/etcd-io/etcd/issues/15360

&nbsp;

2) Additionally this pull request includes documentation for tls min/max options which were recently backported to 3.5 and 3.4 so need to be added to the operations guide.

Refer:
- https://github.com/etcd-io/etcd/pull/15483
- https://github.com/etcd-io/etcd/pull/15486

### CODE REVIEW

**Review by ahrtr — APPROVED** [review]
LGTM

Thank you @jmhbnz

**Comment by jmhbnz** [comment]
Hey @ahrtr - I've included the 3.5 op guide change here for tls min/max options.  Let me know if that's ok, I can separate it if necessary.

**Comment by ahrtr** [comment]
> Hey @ahrtr - I've included the 3.5 op guide change here for tls min/max options. Hope that's ok, I can separate it if necessary.

It isn't a big deal. 

cc @spzala

**Comment by jmhbnz** [comment]
Note to maintainers to hold off merging this until [3.4] Backport tls 1.3 support etcd#15486 merges.

**Comment by spzala** [comment]
> Note to maintainers to hold off merging this until [etcd-io/etcd#15486](https://github.com/etcd-io/etcd/pull/15486) merges.
> 
> Edit: This is now ready to merge, @spzala can you please take a look, thanks!

@jmhbnz I was out of office but back now..reviewing now. Thanks!

**Review by spzala — APPROVED** [review]
Thanks @jmhbnz

**Pull request merged**
